.github/workflows/continuous-integration-prototypes.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       run: Get-ChildItem -Recurse -Include *.sln | ForEach-Object {dotnet build $_ --configuration Release --no-restore; if ($LASTEXITCODE -ne 0 ) {throw "Build for $_ FAILED"; }} 
 
     - name: Test .Net
-      run: Get-ChildItem -Recurse -Include *.sln | ForEach-Object {dotnet test $_ --no-restore --verbosity normal --collect:"XPlat Code Coverage"}  
+      run: Get-ChildItem -Recurse -Include *.sln | ForEach-Object {dotnet test $_ --configuration Release --no-restore --verbosity normal --collect:"XPlat Code Coverage"}  
```

Comment:
Nice catch! :)

Reply:
You might need to PR this one liner separately though because github actions on the main repo are always executed based on what is merged into the main repo's master branch.

Reply:
Sorry, never mind, I checked what's being executed for the PR.
It seemed to pick up the changes you've pushed.
There are other circumstances where you need to fix the workflow separately but here it seems to take effect already.